vcpkg/triplets/arm-android.cmake

```diff
@@ -4,6 +4,18 @@ set(VCPKG_LIBRARY_LINKAGE static)
 set(VCPKG_CMAKE_SYSTEM_NAME Android)
 set(VCPKG_BUILD_TYPE release)
 
-set(ENV{VCPKG_ANDROID_NATIVE_API_LEVEL} "detect")
-set(ENV{CXXFLAGS} "-fstack-protector-strong")
-set(ENV{CFLAGS} "-fstack-protector-strong")
+set(ENV{CXXFLAGS} "-fstack-protector-strong -fsanitize=address -fno-omit-frame-pointer")
+set(ENV{CXXFLAGS_RELEASE} "-fstack-protector-strong -fsanitize=address -fno-omit-frame-pointer")
+set(ENV{CFLAGS} "-fstack-protector-strong -fsanitize=address -fno-omit-frame-pointer")
+set(ENV{CFLAGS_RELEASE} "-fstack-protector-strong -fsanitize=address -fno-omit-frame-pointer")
+set(ENV{LDFLAGS} "-fsanitize=address")
+set(ENV{LDFLAGS_RELEASE} "-fsanitize=address")
+
+set(VCPKG_CXX_FLAGS "-fstack-protector-strong -fsanitize=address -fno-omit-frame-pointer")

```

Comment:
I think we need:

VCPKG_CXX_FLAGS, VCPKG_C_FLAGS, VCPKG_LINKER_FLAGS and could drop the rest.

Reply:
@m-kuhn , done.